egs/polyphone_swiss_french/asr1/local/data_prep.py

```diff
@@ -100,7 +100,7 @@ def _cleantext(self, text):
 
         text = text.lower()
         text = re.sub(r"\[\\?hésitation", " ", text)
-        text = re.sub(r"\[\\?prononciation bizarre", " ", text)
+        text = re.sub(r"\[\\?pronunciation bizarre", " ", text)
```

Comment:
This might be true. 
@AdolfVonKleist, could you check?

Reply:
@sw005320 In this pull request, I leave this unchanged ( f1a753c )

Reply:
Thanks.
This is Swiss French and English spell check may cause some issues.
So, we can leave it as it is.